reconcile/test/change_owners/test_change_type_bundle_change_post_processing.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from reconcile.change_owners.change_owners import _aggregate_file_moves
```

Comment:
test private method `_aggregate_file_moves` seems not right, maybe worth move it to a seperate module then make it public as `aggregate_file_moves`

Reply:
you are absolutely right. i will move some parts around `BundleFileChange`, their initialization and aggregation into a dedicated module.